.github/workflows/test_extensions.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        pip install -r requirements-devel.txt
```

Comment:
note -- the difference between the two besides `[full]` and `[devel]` flavors of install is that in `-devel` we install with `pip -e` i.e. for development.  Should be ok, but just worth noting.

Reply:
Note that I changed it to `-devel` in order to get `coverage` installed.